OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -603,4 +603,16 @@ public interface INotifyPlayerDisconnected
 	{
 		void PlayerDisconnected(Actor self, Player p);
 	}
+
+	public interface IDeveloperMode
```

Comment:
Why do we need this interface? Also by the looks of it it should be renamed to IDeveloperCheats or smth like that

Reply:
The trait is called `DeveloperMode` and lives in Mods.Common, so you can't access it from Game.

Reply:
Could we cache the trait lookup where it's used, to avoid having to teach `Game` about the dev mode stuff?

Reply:
Don't believe so. As the Player/Owner of an actor can change. You could listen for the event. But the pattern occurs so often to me this seemed cleanest. 

Ofcourse you could make IDeveloperMode an empty interface. But you have to cast it everywhere. And in most cases the bool values are only read. So i believe this is cleaner.

You should not perceive it as developer mode - as in debug - code. Rather as runtime information part of the engine used in many places. Feature toggles. 

Cheats as a name doesn't really cover it. DeveloperMode is better.

Reply:
`World.LocalPlayer` and `Player.PlayerActor` don't change. Therefore you cache safely cache the trait lookup near the use site. Ownership changes aren't a worry here - we're not dealing with a "regular" actor after all.

Reply:
Then i will revert all changes in this PR and only cache it locally for the selection rendering.